prometheus-to-sd/config/dynamic_source.go

```diff
@@ -18,8 +19,26 @@ const (
 	nameLabel    = "k8s-app"
```

Comment:
nit: This name is a little ambiguous, maybe "componentNameLabel"?

Reply:
corrected